department_employee_test.rb

```diff
@@ -0,0 +1,109 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './departments'
+require './employees'
+
+
+class EmployeeReviewTest <Minitest::Test
+
+  def test_01_department_class_exists
+    assert Department
+  end
+
+  def test_02_employee_class_exist
+    assert Employee
+  end
+
+  def test_03_add_employee_to_department
+    sports = Department.new("Sports")
+    steve = Employee.new("Steve", 25000)
+    sarah = Employee.new("Sarah", 25000)
+    bill = Employee.new("Bill", 25000)
+    assert sports.add_employee(steve)
+    assert sports.roster.include?(steve)
+    assert sports.add_employee(sarah)
+    assert sports.roster.include?(sarah)
+    assert sports.add_employee(bill)
+    assert sports.roster.include?(bill)
+  end
+
+  def test_04_get_employee_name
+    steve = Employee.new("Steve", 25000)
+    assert steve.name == "Steve"
+    sarah = Employee.new("Sarah", 25000)
+    assert sally.name == "Sarah"
+    bill = Employee.new("Bill", 25000)
+    assert bill.name == "Bill"
+  end
+
+  def test_05_get_employee_salary
+    steve = Employee.new("Steve", 25000)
+    assert steve.salary == 25000
```

Comment:
It would probably be better to have different salary numbers here for the three examples (so that you are testing that the salary method doesn't always return 25000)